docs/user-guide/building-assistants.rst

```diff
@@ -856,7 +968,7 @@ We define some intent mappings to make it more intuitive to the user what an int
 Out of scope intent
 """""""""""""""""""
 
-It is good practice to also handle questions you know your users may ask, but you don’t necessarily have a skill
+It is good practice to also handle questions you know your users may ask, but for which you don’t necessarily have an action 
```

Comment:
hm i feel like i already commented on this? Should be "you don't necessarily have a user goal handled" yet or similar

Reply:
right